packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/widgetforyourapp.md

```diff
@@ -42,10 +42,20 @@ When you initialize the Sign-In Widget on your sign-in page, you must configure
         scopes: /*[[${scopes}]]*/ '[scopes]',
     };
 
-    new OktaSignIn(config).showSignInAndRedirect(
-        { el: '#sign-in-widget' },
-        function (res) {}
-    );
+  const signIn = new OktaSignIn({
+       el: '#sign-in-widget',
+       ...config
+  });
+
+   // Search for URL Parameters to see if a user is being routed to the application to recover password
+   var searchParams = new URL(window.location.href).searchParams;
+   signIn.otp = searchParams.get('otp');
+   signIn.state = searchParams.get('state');
+
+   signIn.showSignInAndRedirect()
+    .catch(err => {
+      console.log('Error happen in showSignInAndRedirect: ', err);
```

Comment:
I don't see why not. This is example code, not specifically from the widget itself. Or specifically from some sample either. (Though it may resemble it a lot)